core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1487,7 +1482,7 @@ FROM (
 	FOR UPDATE OF e1
 ) e0
 WHERE e0.id = evm.txes.id
-RETURNING e0.id, e0.nonce`, ErrCouldNotGetReceipt, cutoff, chainID.String())
+RETURNING e0.id, e0.nonce`, ErrCouldNotGetReceipt, latestFinalizedBlockNum, chainID.String())
```

Comment:
Do we want `HAVING max(evm.tx_attempts.broadcast_before_block_num) < $2 (latestFinalizedBlockNum)` to be inclusive now? Previously `$2` was `cutoff`

Reply:
I think inclusive makes sense. Since it would have been finalized at that block num, we want to mark it as errored if we hadn't gotten a receipt yet at that point

Reply:
Great! Fixed here: [`2adcd80` (#13794)](https://github.com/smartcontractkit/chainlink/pull/13794/commits/2adcd80cd653c1d18558b53b06fc547667dbdf97)